### PR TITLE
custom telescope function to access dotfiles wherever you are

### PR DESCRIPTION
**Pull request by Vanderscycle**
I always tinker with my dotfiles and don't like to change dir to do so. As a result I took inspiration from the primeagen and made this lua function.

### CODE REVIEW

**Comment by siduck76** [comment]
I have tested this and doesnt work :(

**Comment by Vanderscycle** [comment]
@siduck76 weird :(
here's what should happen:
[asciicast](https://asciinema.org/a/sgna3kVPz4eVplh5a1eBPk890)

**Comment by ashincoder** [comment]
that is because the function searches for doots direcotory in ~/.config

File: lua/telescope-nvim.lua [hunk 1]
```diff
@@ -67,4 +67,10 @@ M.config = function()
     require("telescope").load_extension("media_files")
 end
 
+function M.search_dootfiles()
+    require('telescope.builtin').find_files {
+        cwd = "~/.config/nvim/",
```
Comment (Vanderscycle) [inline lua/telescope-nvim.lua:72]: @ashincoder @siduck76 ash pointed out that you may need to change the config location.

**Comment by siduck76** [comment]
I have tested it and it doesnt work , also the mapping too 
image

**Comment by Vanderscycle** [comment]
Hmm... Maybe it's not ready. I don't know why it works on my config and not yours as the changes aren't drastic.

**Comment by siduck76** [comment]
> Hmm... Maybe it's not ready. I don't know why it works on my config and not yours as the changes aren't drastic.

yeah I tested your fork of nvchad and it doesnt work too , can you test it your own fork of nvchad? I think you have might've tested your personal config and it could have a different a different config than the PR which works .

**Comment by Vanderscycle** [comment]
That's what happened 😅. I'll close the pr and reopen when I test more tomorrow.

**Comment by Vanderscycle** [comment]
I really need to get better at testing and I'll make effort so that my next prs are tested on the pr branch and not my config branch.

**Pull request closed without merging**